packages/@aws-cdk/aws-amazonmq/package.json

```diff
@@ -23,6 +23,9 @@
       "python": {
         "distName": "aws-cdk.aws-amazonmq",
         "module": "aws_cdk.aws_amazonmq"
+      },
+      "awsdocs": {
+        "title": "AmazonMQ"
```

Comment:
Shouldn't we use words with spaces then?

Reply:
I was also thinking that I want the README's H1 to be used as the doc title

Reply:
I'm game with that. Cannot auto-generate those... What do you think about using the "AWS Marketing" name for those?

The same name that the product has to the public -- `AWS Lambda`, `Amazon S3`, ...
I have mixed feelings on that one...

Reply:
I think the title should be the official marketing name of the service.

Reply:
+1

Reply:
The question was more on whether the `Amazon` / `AWS` prefixes should be there or not...

Reply:
Yes, they should

Reply:
That's what I have right now... They're conveniently located in a JSON file for easier review... Please someone cross-check!